lib/configuration.sh

```diff
@@ -161,47 +161,70 @@ PACKAGE_LIST_DESKTOP="xserver-xorg xserver-xorg-video-fbdev gvfs-backends gvfs-f
 PACKAGE_LIST_DESKTOP_RECOMMENDS="mirage galculator hexchat xfce4-screenshooter network-manager-openvpn-gnome mpv fbi cups-pk-helper \
 	cups geany atril xarchiver"
 
+
+# For minimal build different set of packages is needed
+if [[ $BUILD_MINIMAL == yes  ]]; then
+
+	# Essential packages for minimal build
+	PACKAGE_LIST="bc cpufrequtils device-tree-compiler fping \
+		fake-hwclock psmisc ntp parted sudo linux-base dialog \
+		ncurses-term sysfsutils toilet u-boot-tools unattended-upgrades \
```

Comment:
IMO unattended-upgrades could be left out?

Reply:
Sudo as well, for Debian at least. Sudo is not shipped by default. Or is it compulsorily needed by the script?

Reply:
Unattended-upgrades removed in caf67f8 

Regarding sudo: albeit I am great fan of disabling root account in all ways but single mode, this do not apply perfectly for armbian ecosystem. Thus I am removing it from default config